### PR TITLE
fix: don't try to seek past start or end

### PR DESCRIPTION
**Pull request by ghr-mohammadi**
now + this.options_.seconds > totalDuration ? "replay not working" : "every thing is ok";

### CODE REVIEW

**Comment by ghr-mohammadi** [comment]
now + this.options_.seconds > totalDuration ? "replay not working" : "every thing is ok";

**Comment by ghr-mohammadi** [comment]
k

**Comment by mister-ben** [comment]
Thanks. Makes sense, I just want to double check this plays nice with live streams.

File: src/plugin.js [hunk 1]
```diff
@@ -125,11 +125,12 @@ class SeekButton extends Button {
    */
   handleClick() {
     const now = this.player_.currentTime();
+    const totalDuration = this.player_.duration();
 
     if (this.options_.direction === 'forward') {
-      this.player_.currentTime(now + this.options_.seconds);
+      this.player_.currentTime(Math.min(now + this.options_.seconds, totalDuration));
```
Comment (mister-ben) [inline src/plugin.js:131]:
```suggestion
      let duration = this.player_.duration();

      if (this.player_.liveTracker && this.player_.liveTracker.isLive()) {
        duration = this.player_.liveTracker.seekableEnd();
      }

      this.player_.currentTime(Math.min(now + this.options_.seconds, duration));
```

File: src/plugin.js [hunk 2]
```diff
@@ -125,11 +125,12 @@ class SeekButton extends Button {
    */
   handleClick() {
     const now = this.player_.currentTime();
+    const totalDuration = this.player_.duration();
```
Comment (mister-ben) [inline src/plugin.js:128]:
```suggestion
```

**Review by mister-ben — CHANGES_REQUESTED** [review]
Thanks!. The suggested changes should make sure there's not an attempt to seek past the furthest point in live.

To fix the failing test, it will need a duration set here, e.g. `this.player.duration(100);`

https://github.com/mister-ben/videojs-seek-buttons/blob/d3de062daee9ebc9633a26343c647b1f80fd95a8/test/plugin.test.js#L90

**Comment by mister-ben** [comment]
Thanks for the contribution, I've made a few updates.

**Pull request merged**